src/bed.cpp

```diff
@@ -67,18 +69,33 @@ Attr_ReadValue BedItem::readAttr(AttrTypes_t attr, PropStream& propStream)
 	return Item::readAttr(attr, propStream);
 }
 
-void BedItem::serializeAttr(PropWriteStream& propWriteStream) const
+bool BedItem::unserializeAttr(const tfs::Item& pbItem)
```

Comment:
This is great. I'll put it under the `protobuf` or `flatbuffers` namespace later.